src/MethodesImpl.java

```diff
@@ -0,0 +1,278 @@
+public class MethodesImpl implements PokemonAttacks{
```

Comment:
Deze klasse heb je nergens gebruikt, klopt dat? Ik snap wat je probeert te doen, maar het werkt op deze manier nog niet echt. Het zou wel tof zijn als je hier op een of andere manier een veralgemenisering van de aanval methodes kunt maken, want de meeste aanval methodes zijn bijna hetzelfde, op een paar kleine details na.